Monika After Story/game/script-topics.rpy

```diff
@@ -6619,3 +6619,289 @@ label monika_hair_down:
+        future_date_count = 0
+        no_dating_joke = False
+
+    label .loopstart:
+        pass
+
+    call mas_start_calendar_select_date
+
+    $ selected_date = _return
+
+    if not selected_date or selected_date.date() == first_sesh_raw.date():
+        # no date selected, we assume user wanted to cancel
+        m 2dsc "[player]..."
+        m 2eka "I thought you said I was wrong."
+        menu:
+            m "Are you sure it's not [first_sesh_formal]?"
+            "It's not that date.":
+                if wrong_date_count >= 2:
+                    label .had_enough:
+                        # monika has had enough of your shit
+                        m 2dsc "..."
+                        m 2lfc "We'll do this another time, then."
+
+                        # we're going to reset the conditional to wait
+                        # 30 more days
+                        $ mas_chgCalEVul(30)
+
+                        return "NOPE"
+
+                # otherwise try again
+                m 2dsc "..."
+                m 2tfc "Then pick the correct date!"
+                $ wrong_date_count += 1
+                jump .loopstart
+
+            "Actually that's the correct date. Sorry.":
+                m 2eka "That's okay."
+                $ selected_date = first_sesh_raw
+
+    elif selected_date.date() == datetime.date.today():
+        # today was chosen
+        if wrong_date_count >= 2:
+            jump .had_enough
+
+        m 2dsc "..."
+        m 2tfc "We did {b}not{/b} just start dating today."
+        m "Take this seriously, [player]."
+        $ wrong_date_count += 1
+        jump .loopstart
+
+    elif selected_date > first_sesh_raw:
+        # you selected a future date?! why!
+        if future_date_count > 0:
+            # don't play around here
+            jump .had_enough
+
+        $ future_date_count += 1
+        m 1wud "What..."
+        menu:
+            m "We haven't been dating this whole time?"
+            "That was a misclick!":
+                # relif expression
+                m 1duu "{cps=*2}Oh, thank god.{/cps}"
+
+                label .misclick:
+                    m 2tfc "[player]!"
+                    m 2eua "You had me worried there."
+                    m "Don't misclick this time!"
+                    jump .loopstart
+
+            "Nope.":
+                m 1dsc "..."
+
+                show screen mas_background_timed_jump(5, "monika_dating_startdate_confirm.tooslow")
+
+                menu:
+                    "I'm kidding.":
+                        hide screen mas_background_timed_jump
+                        # wow what a mean joke
+
+                        if no_dating_joke:
+                            # you only get this once per thru
+                            jump .had_enough
+
+                        # otherwise mention that this was mean
+                        m 2tfc "[player]!"
+                        m 2rksdlc "That joke was little mean."
```

Comment:
` +                        m 2rksdlc "That joke was little mean." `
Try "That joke was a little mean."